langchain/prompt.py

```diff
@@ -126,3 +136,110 @@ def from_examples(
         example_str = example_separator.join(examples)
         template = prefix + example_str + suffix
         return cls(input_variables=input_variables, template=template)
+
+
+class DynamicPrompt(BaseModel, BasePrompt):
+    r"""Schema to represent a dynamic prompt for an LLM.
```

Comment:
what is the r for?

Reply:
typo!

Reply:
ah i forgot the linter told me to do this
./langchain/prompt.py:133:1: D301 Use r""" if any backslashes in a docstring